support/rewriteInheritance.js

```diff
@@ -0,0 +1,29 @@
+var through = require('through2');
+
+/**
+ * Replace all instances of `__proto__` in a string with a cross-browser
+ * compatible alternative.
+ *
+ * @param {string} js
+ * @return {string}
+ */
+function parseInheritance(js) {
+  return js.replace(/^ *(\w+)\.prototype\.__proto__ * = *(\w+)\.prototype *;?/gm, function(_, child, parent){
```

Comment:
Completely forgot about that, thought you dug it up somewhere else ;) (heh, my name is even in the edit-list of said file.. oops).

Let's leave this for now :) although a future improvement would definitely be getting rid of these.